app/models/full_refresh_upload_task.rb

```diff
@@ -0,0 +1,52 @@
+class FullRefreshUploadTask < MqttClientTask
+  RESPONSE_FORMAT = "tar".freeze
+
+  def content(_options)
+    payload = CatalogInventory::Payload.new(RESPONSE_FORMAT, upload_url, full_refresh_jobs)
+    self[:input] = payload.as_json
+
+    payload.as_json
+  end
+
+  def upload_url
+    ENV.fetch("UPLOAD_URL") || raise("UPLOAD_URL must be specified")
+  end
+
+  def full_refresh_jobs
```

Comment:
This method seems too fat. Consider to break into multiple methods. Then it looks like
```
job << new_job_template_job
job << new_credential_job
...
```